### PR TITLE
Feat/display address

### LINKED ISSUE #78: Display address in QR Code page
**Describe the feature**
Display address in QR Code page (if location exists).
image

**Additional context**
Related StackOverflow post: https://stackoverflow.com/questions/9409195/how-to-get-complete-address-from-latitude-and-longitude

### PR DESCRIPTION
**Pull request by harryz6560**
Fixes #78 

# Description

- added horizontal recycler view to show all address
- shows correct address from location
- method implemented in QRCodeViewModel
# Screenshot 

Screenshot 2023-04-01 143442
-the vid shows the scrolling works

[device-2023-04-01-132726.webm](https://user-images.githubusercontent.com/56093344/229313054-63c7eb6c-239e-4d44-86ff-52b18e5f2164.webm)

# Additional Context

N/A

### CODE REVIEW

**Comment by Siddharth-Chhatbar** [comment]
looks good

File: QRHunter/app/src/main/java/com/example/qrhunter/ui/qrcode/QrCodeViewModel.java [hunk 1]
```diff
@@ -10,22 +14,16 @@
...
     private final MutableLiveData<ArrayList<Comment>> comments = new MutableLiveData<>(new ArrayList<>());
     private QRCodeRepository qrCodeRepository = new QRCodeRepository();
     private CommentRepository commentRepository = new CommentRepository();
 
-    /**
-     * Gets the QR code with the specified ID.
-     *
-     * @param qrCodeId The ID of the QR code to get.
-     * @return The QR code to be returned.
-     */
```
Comment (MaxxonTan) [inline QRHunter/app/src/main/java/com/example/qrhunter/ui/qrcode/QrCodeViewModel.java:28]: Curious why these comments are removed. Did u rewrite them somewhere else?

**Review by MaxxonTan — CHANGES_REQUESTED** [review]
looks 🔥🔥. Just a small comment

File: QRHunter/app/src/main/java/com/example/qrhunter/ui/qrcode/QrCodeViewModel.java (hunk 1 above)
Comment (harryz6560) [reply QRHunter/app/src/main/java/com/example/qrhunter/ui/qrcode/QrCodeViewModel.java:28]: oh i'll add them back i think i accidentally over wrote it during merging

**Review by MaxxonTan — APPROVED** [review]

**Comment by MaxxonTan** [comment]
@harryz6560 Feel free to merge when u're ready

**Review by lkaijie — APPROVED** [review]
looks good gamer

**Pull request merged**